global.json

```diff
@@ -1,11 +1,12 @@
 {
   "sdk": {
-    "version": "5.0.100-alpha1-015536"
+    "version": "5.0.100-alpha1-015723"
   },
   "tools": {
-    "dotnet": "5.0.100-alpha1-015752",
+    "dotnet": "5.0.100-alpha1-015723",
     "runtimes": {
-      "dotnet/x64": [
+      "dotnet": [
+        "3.1.0",
```

Comment:
How does this help? This change implies our customers will be unable to build applications mixing EF with our 5.0 release without similar shenanigans -- not a great experience.

Reply:
This is for running dotnet-ef specifically. Not sure what the story for that is going to be @ajcvickers @bricelam ?

Reply:
Hmm, why doesn't the tool roll forward on **major** version?

Reply:
dotnet-ef will eventually target netcoreapp5.0